test/metabase/driver/sql_jdbc/sync/describe_table_test.clj

```diff
@@ -148,7 +148,7 @@
                (:fields (driver/describe-table driver db table))))))
 
 (deftest database-types-fallback-test
-  (mt/test-drivers (sql-jdbc-drivers-using-default-describe-table-or-fields-impl)
+  (mt/test-drivers (disj (sql-jdbc-drivers-using-default-describe-table-or-fields-impl) :druid-jdbc)
```

Comment:
Hardcoding driver names in tests is a no-no, see https://github.com/metabase/metabase/issues/26807 and https://github.com/metabase/metabase/pull/41384

The main reason is that any third-party driver authors that would maybe need to do the same thing have to way of doing so.

I think the best solution going forward is to define namespaced features next to the test, 

https://github.com/metabase/metabase/blob/856b449/test/metabase/query_processor_test/timezones_test.clj#L250-L263

is an example of how to do this without making #26807 any worse

Reply:
I like the idea of enabling tests per feature supported by the tested driver. The proposed approach (Snowflake) works for tests that behave differently based on the driver capabilities -- we not opt out of testing completely.

Here however test should not even start for Druid JDBC. To the best of my knowledge (if I don't want to do `(when (driver/database-supports? DRIVER FEATURE DATABASE) TEST-CODE-TO-RUN)`), construct for that in the test tooling we have is `mt/test-driver`(s).

Then we have `mt/normal-drivers-with-feature` returning set of _normal_ drivers. This is also dependent on `mt/db` call. Before `default-dataset`, introduced by this PR, it could be problematic to use with some drivers.

So imo `database-supports?` is a great way to determine change of behavior of test that we intent to run with the current driver (and not skip it). Also will probably make us dependent on actually loaded (as per `tx/create-db`) that would cause problems for Druids prior to this PR.

Hence I've decided to use `timeseries-drivers` here and I'm further thinking of better approach.